### PR TITLE
[EMCAL-1037] Task for EMCAL BC monitoring

### PR DESCRIPTION
**Pull request by mfasDa**
Monitoring BC distributions for
- EMCAL readout (physics triggers only)
- CTP digits for all EMCAL triggers (including MB readout trigger) Needs CTP readout in run

### CODE REVIEW

**Comment by mfasDa** [comment]
Must be a dedicated task for two reasons:
- Requires CTP readout, which is not guaranteed in all runs (in particular not in stand-alone runs)
- Should not be downsampled, processing of CTP digits and trigger records is fast

**Comment by mfasDa** [comment]
Ready to merge

**Comment by mfasDa** [comment]
CI errors unrelated

**Comment by mfasDa** [comment]
CI errors unrelated

File: Modules/EMCAL/src/BCTask.cxx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+#include <TCanvas.h>
+#include <TH1.h>
+
+#include <boost/algorithm/string/case_conv.hpp>
+
+#include "CommonConstants/LHCConstants.h"
+#include "CommonConstants/Triggers.h"
+#include "DataFormatsCTP/Configuration.h"
+#include "DataFormatsCTP/Digits.h"
+#include "DataFormatsEMCAL/TriggerRecord.h"
```
Comment (knopers8) [inline Modules/EMCAL/src/BCTask.cxx:26]: since these come from O2, `<>` brackets could be used instead.

**Review by knopers8 — APPROVED** [review]

**Pull request merged**